superset/queries/dao.py

```diff
@@ -59,6 +59,8 @@ def update_saved_query_exec_info(query_id: int) -> None:
     def save_metadata(query: Query, payload: Dict[str, Any]) -> None:
         # pull relevant data from payload and store in extra_json
         columns = payload.get("columns", {})
+        for col in columns:
+            col["column_name"] = col.pop("name")
```

Comment:
from the query execution the dict comes back `column[name]`, and so decided to update the naming here before returning it to the client